hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,83 +10,145 @@ This module conforms to the [Comportable guideline for peripheral functionality.
 
 ## Features
 
-- This revision supports a LFSR (linear feedback shift register) digital source, and provides an interface to an external rng (random noise generator) source.
-The LFSR is used as a digital, pseudo-random type of entropy source, while the rng external source is a true entropy source.
-A noise source and its relation to an entropy source are defined by [SP 800-90B](https://csrc.nist.gov/publications/detail/sp/800-90b/final).
-- A set of registers is provided for firmware to manage and obtain entropy bits.
-- The set of registers is designed such that firmware can select between a digital or an rng entropy source.
+- This revision provides an interface to an external physical random noise generator (also referred to as a physical true random number generator or PTRNG) source, and an LFSR (linear feedback shift register) digital source.
+The LFSR is used as a digital, pseudo-random type of entropy source, while the PTRNG external source is a physical binary noise source.
```

Comment:
Is "binary" accepted NIST jargon here?  I don't know well enough to be sure, just want someone to ack/double-check it.